### PR TITLE
Add 112's Summer Tournament 2020 article

### PR DESCRIPTION
**Pull request by zeusminus**
- Add 112's Summer Tournament 2020 wiki article. 
- Add podium, badge, and banner images.
- Update `Tournaments` article accordingly.

### CODE REVIEW

File: wiki/Tournaments/112/Summer_2020/en.md [hunk 1]
```diff
@@ -0,0 +1,545 @@
+---
+tags:
+- 112servis
+- 112ST
+- 112ST 2020
```
Comment (TPGPL) [inline wiki/Tournaments/112/Summer_2020/en.md:5]:
```suggestion
- 112ST2020
```

File: wiki/Tournaments/112/Summer_2020/en.md [hunk 2]
```diff
@@ -0,0 +1,545 @@
...
+---
+
+# 112's Summer Tournament 2020
+
+![112ST2020 Banner](img/banner.png "112's Summer Tournament 2020")
```
Comment (TPGPL) [inline wiki/Tournaments/112/Summer_2020/en.md:10]:
```suggestion
![112ST2020 banner](img/banner.png "112's Summer Tournament 2020")
```

File: wiki/Tournaments/112/Summer_2020/en.md [hunk 3]
```diff
@@ -0,0 +1,545 @@
...
+
+The **112's Summer Tournament 2020** (***112ST2020***) was a double-elimination 1v1 osu! tournament hosted by ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470) and ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852). The tournament was only allowed for Turkish-speaking users regardless of their country flags. This was the sixth installment of 112's Tournaments series.
```
Comment (TPGPL) [inline wiki/Tournaments/112/Summer_2020/en.md:12]:
```suggestion
The **112's Summer Tournament 2020** (***112ST2020***) was a double-elimination 1v1 osu! tournament hosted by ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470) and ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852). The tournament was only allowed for Turkish-speaking users regardless of their country flags. It was the 6th installment of 112's Tournaments series.
```

File: wiki/Tournaments/112/Summer_2020/en.md [hunk 4]
```diff
@@ -0,0 +1,545 @@
...
+| Finals | 2020-09-19/2020-09-23 |
+| Grand Finals | 2020-09-26/2020-09-27 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 4 months of osu!supporter tag |
+| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag |
```
Comment (TPGPL) [inline wiki/Tournaments/112/Summer_2020/en.md:33]:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 4 months of osu!supporter tag |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter tag |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag |
```

File: wiki/Tournaments/112/Summer_2020/en.md [hunk 5]
```diff
@@ -0,0 +1,545 @@
...
+
+![112ST2020 Winner Badge](img/badge.png "112's Summer Tournament 2020 Winner Badge")
```
Comment (TPGPL) [inline wiki/Tournaments/112/Summer_2020/en.md:35]:
```suggestion
![112ST2020 winner badge](img/badge.png "112's Summer Tournament 2020 winner badge")
```

File: wiki/Tournaments/112/Summer_2020/en.md [hunk 6]
```diff
@@ -0,0 +1,545 @@
...
+| Commentator | ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [Coderbora](https://osu.ppy.sh/users/11873333), ![][flag_TR] [Cynesta](https://osu.ppy.sh/users/8674027), ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852) |
+| Designer | ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852) |
+| Website developer | ![][flag_TR] [Coderbora](https://osu.ppy.sh/users/11873333), ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852) |
+| Statistician | ![][flag_TR] [Coderbora](https://osu.ppy.sh/users/11873333) |
+| Wiki editor | ![][flag_TR] [Zeus-](https://osu.ppy.sh/users/5464437) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1097571)
+- [Challonge link](https://challonge.com/112yaz2020)
```
Comment (TPGPL) [inline wiki/Tournaments/112/Summer_2020/en.md:56]:
```suggestion
- [Challonge](https://challonge.com/112yaz2020)
```

File: wiki/Tournaments/112/Summer_2020/en.md [hunk 7]
```diff
@@ -0,0 +1,545 @@
...
+- [112's Tournaments Discord server](https://discord.gg/g3nqqeZ)
+- [Livestream](https://www.twitch.tv/osu_turkiye)
+- [Official website](https://112ninturnuvalari.xyz/kurallar)
+
+## Participants
+
+| Ranking | Players | Qualifier points |
```
Comment (TPGPL) [inline wiki/Tournaments/112/Summer_2020/en.md:63]:
```suggestion
| Ranking | Player | Qualifier points |
```

File: wiki/Tournaments/112/Summer_2020/en.md [hunk 8]
```diff
@@ -0,0 +1,545 @@
...
+| #23 | ![][flag_TR] [Belif0226](https://osu.ppy.sh/users/10585565) | 21.66 |
+| #24 | ![][flag_DE] [Skellexon](https://osu.ppy.sh/users/11268328) | 21.77 |
+| #25 | ![][flag_NL] [QuadroKiller](https://osu.ppy.sh/users/7552072) | 24.55 |
+| #26 | ![][flag_TR] [cANITIN](https://osu.ppy.sh/users/11717778) | 25 |
+| #27 | ![][flag_TR] [nitromehmet](https://osu.ppy.sh/users/9909537) | 25.55 |
+| #28 | ![][flag_TR] [selimax](https://osu.ppy.sh/users/13525233) | 27.44 |
+
+## Podium
+
+![112ST2020 Podium](img/podium.png "112's Summer Tournament Podium")
```
Comment (TPGPL) [inline wiki/Tournaments/112/Summer_2020/en.md:96]:
```suggestion
![112ST2020 podium](img/podium.png "112's Summer Tournament podium")
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/112/Summer_2020/en.md [hunk 9]
```diff
@@ -0,0 +1,545 @@
...
+- 112servis
+- 112ST
+- 112ST2020
+---
+
+# 112's Summer Tournament 2020
+
+![112ST2020 banner](img/banner.png "112's Summer Tournament 2020")
+
+The **112's Summer Tournament 2020** (***112ST2020***) was a double-elimination 1v1 osu! tournament hosted by ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470) and ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852). The tournament was only allowed for Turkish-speaking users regardless of their country flags. It was the 6th installment of 112's Tournaments series.
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2020/en.md:12]:
```suggestion
The **112's Summer Tournament 2020** (***112ST2020***) was a double-elimination 1v1 osu! tournament hosted by ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470) and ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852). The tournament was only allowed for Turkish-speaking users regardless of their country flags. It was the sixth installment of 112's Tournaments series.
```

File: wiki/Tournaments/112/Summer_2020/en.md [hunk 10]
```diff
@@ -0,0 +1,545 @@
...
+| Commentator | ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [Coderbora](https://osu.ppy.sh/users/11873333), ![][flag_TR] [Cynesta](https://osu.ppy.sh/users/8674027), ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852) |
+| Designer | ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852) |
+| Website developer | ![][flag_TR] [Coderbora](https://osu.ppy.sh/users/11873333), ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852) |
+| Statistician | ![][flag_TR] [Coderbora](https://osu.ppy.sh/users/11873333) |
+| Wiki editor | ![][flag_TR] [Zeus-](https://osu.ppy.sh/users/5464437) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1097571)
+- [Challonge](https://challonge.com/112yaz2020)
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2020/en.md:56]:
```suggestion
- [Challonge brackets](https://challonge.com/112yaz2020)
```

File: wiki/Tournaments/112/Summer_2020/en.md [hunk 11]
```diff
@@ -0,0 +1,545 @@
...
+| --: | :-: | :-: | :-- | :-- |
+| **[heyronii](https://osu.ppy.sh/users/5642779)** | **6** | 1 | [Mikatsu](https://osu.ppy.sh/users/6612642) | [#1](https://osu.ppy.sh/community/matches/67191627) |
+
+### Losers Round 4 
+
+Saturday, 19 September 2020
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| [CutieDemon](https://osu.ppy.sh/users/6440158) | 0 | **1** | **[emptypudding](https://osu.ppy.sh/users/11003085)** | *-win by default-* |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2020/en.md:306]:
```suggestion
| [CutieDemon](https://osu.ppy.sh/users/6440158) | 0 | **1** | **[emptypudding](https://osu.ppy.sh/users/11003085)** | *win by default* |
```
Do for others too 👀

**Review by fajar13k — COMMENTED** [review]
Minor

File: wiki/Tournaments/112/Summer_2020/en.md (hunk 9 above)
Comment (zeusminus) [reply wiki/Tournaments/112/Summer_2020/en.md:12]: Well, @TPGPL requested to change that one before in [one of his suggestions](https://github.com/ppy/osu-wiki/pull/4256#discussion_r495589653). Why do you want to revert it now? I'm confused.

Comment (TPGPL) [reply wiki/Tournaments/112/Summer_2020/en.md:12]: looks like the formatting for this sentence is pretty inconsistent, as tournaments have it both way (e.g. OWC uses the number format for its installments while TWC uses text format)

**Review by TPGPL — APPROVED** [review]

**Pull request merged**